spec/roman_numerals_spec.rb

```diff
@@ -39,4 +44,64 @@
       result = arabic_to_roman.convert(8)
       expect(result).to eq "VIII"
     end
+
+    it 'returns IX for 9' do
+	  result = arabic_to_roman.convert(9)
+	  expect(result).to eq "IX"
+    end
+
+    it 'returns X for 10' do
+	  result = arabic_to_roman.convert(10)
+	  expect(result).to eq "V"
+    end
+
+    it 'returns XI for 11' do
+	  result = arabic_to_roman.convert(11)
+	  expect(result).to eq "XI"
+    end
+
+    it 'returns XII for 12' do
+	  result = arabic_to_roman.convert(12)
+	  expect(result).to eq "XII"
+    end
+
+    it 'returns XIII for 13' do
+	  result = arabic_to_roman.convert(13)
+	  expect(result).to eq "XIII"
+    end
+
+    it 'returns XIV for 14' do
+	  result = arabic_to_roman.convert(14)
+	  expect(result).to eq "XIV"
+    end
+
+    it 'returns XV for 15' do
+	  result = arabic_to_roman.convert(15)
+	  expect(result).to eq "XV"
+    end
+
+    it 'returns XVI for 16' do
+	  result = arabic_to_roman.convert(16)
+	  expect(result).to eq "XVI"
+    end
+
+    it 'returns XVII for 17' do
+	  result = arabic_to_roman.convert(17)
+	  expect(result).to eq "XVII"
+    end
+
+    it 'returns XVIII for 18' do
+	  result = arabic_to_roman.convert(18)
+	  expect(result).to eq "XVIII"
+    end
+
+    it 'returns XIX for 19' do
+	  result = arabic_to_roman.convert(19)
+	  expect(result).to eq "XIX"
+    end
+
+    it 'returns XX for 20' do
+	  result = arabic_to_roman.convert(20)
+	  expect(result).to eq "XX"
+    end
```

Comment:
I don't think these tests will pass.
Please don't commit to the repo if any tests are failing.